cwltool/cwlviewer.py

```diff
@@ -0,0 +1,119 @@
+import rdflib
+import pygraphviz as pgv
+from urllib.parse import urlparse
+import os
+
+
+class CWLViewer:
+    """
+    Visualize a CWL workflow. The viewer tagrets to produce similar images with the
+    https://github.com/common-workflow-language/cwlviewer.
+    """
+
+    _queries_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'rdfqueries')
+    _get_inner_edges_query_path = os.path.join(_queries_dir, 'get_inner_edges.sparql')
+    _get_input_edges_query_path = os.path.join(_queries_dir, 'get_input_edges.sparql')
+    _get_output_edges_query_path = os.path.join(_queries_dir, 'get_output_edges.sparql')
+    _get_root_query_path = os.path.join(_queries_dir, 'get_root.sparql')
+
+    def __init__(
+            self,
+            rdf_description  # type: str
+    ):
+        self._dot_graph = CWLViewer._init_dot_graph()  # type: pgv.agraph.AGraph
+        self._rdf_graph = self._load_cwl_graph(rdf_description)  # type: rdflib.graph.Graph
+        self._root_graph_uri: str = self.get_root_graph_uri()
```

Comment:
```suggestion
        self._root_graph_uri = self.get_root_graph_uri()  # type: str
```

We still have to support Python 3.5 for a little while longer :-)

Reply:
Sure! sorry! my mistake! I will fix it